doc/api/events.md

```diff
@@ -508,7 +508,21 @@ added: v0.1.26
 
 Removes all listeners, or those of the specified `eventName`.
 
-It is bad practice to remove listeners added elsewhere in the code,
+```js
+server
+.on('first_connection', () => {
+  console.log('log first connection');
+})
+.on('second_connection', () => {
+  console.log('log second connection');
+});
+// Passing a name event removes that one.
+server.removeAllListeners('first_connection');
+// No passing any param removes all the listeners
+server.removeAllListeners();
+```
+
+Note that it is bad practice to remove listeners added elsewhere in the code,
```

Comment:
Welcome @jshricarde and thanks for the pull request! Please don't add "Note that" to this sentence. (Our linter calls that out. Try `make lint-md`.) Thanks!

Reply:
Oops! Correction! Our linter does not call that out! But it will very soon. There's an open PR for it.

Reply:
```suggestion
It is bad practice to remove listeners added elsewhere in the code,
```